### ISSUE TITLE
Sequence Diagrams with messageAlign smash text with <br>

### ISSUE THREAD

**Issue by kevinawoo** [mermaid-js/mermaid-cli#349]
**Describe the bug**
This issue reported in https://github.com/mermaid-js/mermaid/issues/3011 still exists on the cli at `v9.1.5-beta.8` when creating `png`s, 

I noticed it was resolved in the v9.1.5 of mermaid.js though. [Live Editor example, currently at v9.1.5](https://mermaid-js.github.io/mermaid-live-editor/edit#pako:eNpVkEFrwzAMhf-KMIRc2qYtO5lR6LqxwgYbNLv54jlqYmbLmaNsLSH_fd7aQneT3vf0JDQIEyoUUmTZYMmyhAHyDj97JIN56nKPXadrXDtbUxJyh3vORxjHLFN0cd5bXUftFb11GKer1V0M36mSsHHWfECDzgVFZzXxHcavX_yIDMUfhW1ZvhaL2QJu3-MKtqFLt-BB-9bhzAR_ktfGYJsA44GLhr1TdIq63nlJKmA5n8PL02l0E4iReFoeW7wK-A-fkWpuJNwsFYmJ8Bi9tlV6z6AIQAlu0KMSMpUV7nXvWAlFY7L2baUZHyrLIQq5167DidA9h92RjJAce7yYzs86u8YfFrWDDA)

**To Reproduce**

1. create a markdown file with the same contents as https://github.com/mermaid-js/mermaid/issues/3011:
	````md
	```mermaid
	%%{init: { 'sequence': {'messageAlign': 'left'} }}%%
	sequenceDiagram
	User->>Browser: Click hello
	Browser->>Server: Get /hello HTTP/1.1 <br> Host: example.com <br> Accept: text/html
	Server->>Browser: HTTP/1.1/ 200 OK <br> Content-Type: text/html <br> Content-Length: 42
	```
	````
2. Attempt to generate a PNG. Here's the example of I've been doing
	  ```bash
	  docker run -v $(pwd):/data ghcr.io/mermaid-js/mermaid-cli/mermaid-cli:9.1.5-beta.8 --scale 2 -i /data/./test.md -o /data/./test.png
	  ```
3. Noticed the generated image has the `message` text between `Browser` and `Server` is overlayed on top of each other.
	<kbd><img width="687" alt="image" src="https://user-images.githubusercontent.com/3469532/167208707-b659d643-daad-49e6-8511-7cd42488c597.png"></kbd>

**Expected behavior**
![image](https://user-images.githubusercontent.com/3469532/167208473-c8599c07-75d6-40df-9155-bbf4599016ee.png)

**Desktop (please complete the following information):**

- OS: docker
- version: 9.1.5-beta.8

**Comment by aloisklink** [comment]
Seems to be fixed in mermaid-cli 9.1.5 (released yesterday)

Try using `ghcr.io/mermaid-js/mermaid-cli/mermaid-cli:9.1.5` instead of `9.1.5-beta.8`.

Unfortunately, it looks like some of the text is getting cut on the right side of the screen. However, since the same thing happens on the [Mermaid Live Editor](https://mermaid-js.github.io/mermaid-live-editor/edit#pako:eNpVkEFrwzAMhf-KMIRc2qYtO5lR6LqxwgYbNLv54jlqYmbLmaNsLSH_fd7aQneT3vf0JDQIEyoUUmTZYMmyhAHyDj97JIN56nKPXadrXDtbUxJyh3vORxjHLFN0cd5bXUftFb11GKer1V0M36mSsHHWfECDzgVFZzXxHcavX_yIDMUfhW1ZvhaL2QJu3-MKtqFLt-BB-9bhzAR_ktfGYJsA44GLhr1TdIq63nlJKmA5n8PL02l0E4iReFoeW7wK-A-fkWpuJNwsFYmJ8Bi9tlV6z6AIQAlu0KMSMpUV7nXvWAlFY7L2baUZHyrLIQq5167DidA9h92RjJAce7yYzs86u8YfFrWDDA), I'm guessing it's a mermaid bug.

![test-1](https://user-images.githubusercontent.com/19716675/184962002-cc6ce001-8d1e-404c-a9c8-8c9e00dc8768.png)

By the way, if want that activation box, you can use [`activate Server`](https://mermaid-js.github.io/mermaid/#/sequenceDiagram),

e.g.

````md
```mermaid
%%{init: { 'sequence': {'messageAlign': 'left'} }}%%
sequenceDiagram
User->>Browser: Click hello
Browser->>Server: Get /hello HTTP/1.1 <br> Host: example.com <br> Accept: text/html
activate Server
Server->>Browser: HTTP/1.1/ 200 OK <br> Content-Type: text/html <br> Content-Length: 42
deactivate Server
```
````

![test-1](https://user-images.githubusercontent.com/19716675/184963002-5f4a2d86-003b-4306-8ce3-21fcbdc2e15f.png)

**Comment by kevinawoo** [comment]
Since `9.1.5` works for me, going to close this. Thanks!
